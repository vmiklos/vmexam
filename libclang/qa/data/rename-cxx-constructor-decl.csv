C::nX,m_nX
C::aA,m_aA
