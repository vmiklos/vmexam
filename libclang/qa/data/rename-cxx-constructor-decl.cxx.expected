class A
{
};

class C
{
public:
    C();
    int m_nX;
    A m_aA;
};

C::C()
     : m_nX(0)
{
}
